### DISCUSSION TITLE
I want fine-grained token that lives forever

### DISCUSSION THREAD

**Discussion by KSXGitHub** [community/community#113661] [Repositories]
### Select Topic Area

Product Feedback

### Body

Having to spend hours to figure out why CI breaks is not a pleasant experience. So until fine-grained token can live forever, I'm stuck with classics.

**Comment by IliqNikushev** [comment]
did you try to set them for a LONG period and running a dummy call to check if they are still working? In general it is good practice to rotate the tokens just in case if something got leaked
I created a topic to see if it would be possible to programmatically work with them https://github.com/orgs/community/discussions/120437
